### PR TITLE
[Tests-Only] Created versionsPanel PO & moved related commands/elements

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
1. Separate page-object for versions dialog box is created
2. related commands/elements are placed in versionsDialog

## Related Issue
- Part of #2677

## Motivation and Context
Appropriate placements for different PO methods and elements

## How Has This Been Tested?

- :rabbit: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  commands: {
+    getVersions: async function (filename) {
```
Comment (individual-it) [inline tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js]: that should not depend on the filename, the versions dialog PO should not have to open itself

Comment (kiranparajuli589) [reply tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js]: done!

File: tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  commands: {
+    /**
+     * @returns {*}
+     */
+    openVersionsSidebar: function () {
```
Comment (individual-it) [inline tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js]: the versions tab cannot open itself, should be a part of the side-bar page object, also I would propose `openVersionsTab` as name

Comment (kiranparajuli589) [reply tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js]: done!

**Pull request merged**